### ISSUE TITLE
:farmer: Integration tests failing in gz-gui main 

### ISSUE THREAD

**Issue by Crola1702** [gazebosim/gz-gui#562]
## Environment
* OS Version: Ubuntu 22.04
* Source gz-gui main build

## Description
* Expected behavior: Tests should pass
* Actual behavior: Test are failing

## Steps to reproduce
1. Run a build in https://build.osrfoundation.org/job/gz_gui-ci-main-jammy-amd64/
2. See 2 integration tests fail

## Output

Reference build: https://build.osrfoundation.org/job/gz_gui-ci-main-jammy-amd64/6/

Test regressions:
* [INTEGRATION_minimal_scene.test_ran](https://build.osrfoundation.org/job/gz_gui-ci-main-jammy-amd64/6/testReport/junit/(root)/INTEGRATION_minimal_scene/test_ran/)
* [INTEGRATION_transport_scene_manager.test_ran](https://build.osrfoundation.org/job/gz_gui-ci-main-jammy-amd64/6/testReport/junit/(root)/INTEGRATION_transport_scene_manager/test_ran/)

Log output:
```
55: [ RUN      ] MinimalSceneTest.Config
55: [GUI] [Wrn] [Application.cc:932] [QT] QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-jenkins'
55/58 Test #55: INTEGRATION_minimal_scene ...................***Exception: SegFault  1.42 sec
```

```
57: [ RUN      ] TransportSceneManagerTest.Config
57: [GUI] [Wrn] [Application.cc:932] [QT] QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-jenkins'
57/58 Test #57: INTEGRATION_transport_scene_manager .........***Exception: SegFault  2.94 sec
```

This test has been failing consistently in the last 2 weeks

**Comment by azeey** [comment]
I think these should be fixed by #561

**Comment by mjcarroll** [comment]
Fixed by #561
